libnavigation-base/build.gradle

```diff
@@ -19,6 +19,10 @@ dependencies {
     api dependenciesList.mapboxSdkGeoJSON
 
     implementation dependenciesList.supportAnnotation
+    api dependenciesList.coroutinesCore
```

Comment:
See comments above re: `coroutinesAndroid` and `1.3.3`

Reply:
👍